Blog Post: How do you get Strimzi container images? (#522)
strimzi/strimzi.github.io: _posts/2025-08-12-how-do-you-get-Strimzi-container-images.md

So properly testing the container images is really important.

All of this applies also when you have private fork of Strimzi with some custom patches.
But if that is what you are doing, you are one of the most advanced Strimzi users, and probably already know all of this.

### Vendors

If all of the above seems like a lot of effort, you can consider one of the vendors who have products based on Strimzi.
Typically, the vendors will have dedicated teams with people who have Strimzi know-how and thanks to that will be able to help you even during the holiday season.
They will also have their own build pipelines, run the system tests for you after every build, use their own container registries with very high availability, and so on.
A good vendor will essentially do all the stuff I talked about here for you.

However, the vendors usually do not do this for free.
You might need to buy some license or subscription.
So make sure you understand the exact conditions under which you can use their products.

### Conclusion

As promised at the beginning of the blog post, we went through the different methods how to get the Strimzi container images.

> And we covered the different issues and risks each of these methods might have.

Reviewer: ```suggestion
And we covered the issues and risks each of each method.
```

Unsurprisingly, the easier way to get the binaries, the more risks and issues it poses.

> And if you want some better solution, it might also mean more work from your side.

Reviewer: ```suggestion
And if you want a more robust and secure solution, it might also mean more work from your side.
```
